src/Entity/EntityTypeManager.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Retrofit\Drupal\Entity;
+
+use Drupal\Core\Entity\EntityTypeInterface;
+
+class EntityTypeManager extends \Drupal\Core\Entity\EntityTypeManager
+{
+    /**
+     * @return EntityTypeInterface[]
+     */
+    protected function findDefinitions(): array
+    {
+        $definitions = $this->getDiscovery()->getDefinitions();
+        if (isset($definitions['user_role']) && $definitions['user_role'] instanceof EntityTypeInterface) {
+            $definitions['user_role']->setClass(Role::class);
+        }
```

Comment:
We're overriding this method to allow altering the definition, correct? Which is preventing us from calling `parent::findDefinitions()` and copying all of its code.

We should override `alterDefinitions` and perform this logic there.